gradle.properties

```diff
@@ -2,3 +2,4 @@ org.gradle.caching=true
 org.gradle.configureOnDemand=true
 org.gradle.jvmargs=-Xmx6G
 org.gradle.parallel=true
+kotlin.stdlib.default.dependency=false
```

Comment:
The crucial change: tell Gradle to no longer bundle `kotlin-stdlib` into plugin zip.

Kotlin libraries are provided by IntelliJ anyway! In the same manner, we don't need to bundle e.g. IntelliJ API libraries into our plugin... they're provided OOTB when plugin runs in the IDE